### PR TITLE
Czech locale update

### PR DESCRIPTION
**Pull request by PetrDlouhy**
Update of czech locale + some hardcoded strings added to locales

### CODE REVIEW

File: app/views/notifications/new_group_thread.cs-CZ.text.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+You aren't subscribed to this thread, so you won't receive any more updates.
+You can read the whole thread, and subscribe to it at:
+<%= thread_url(@thread) %>
+
+--------
+You have received this message because your preferences indicate that you
+would like us to notify you on matters in your group.
+
+To stop these notifications, edit your user preferences at:
+<%= edit_user_prefs_url(@member) %>
```
Comment (nikolai-b) [inline app/views/notifications/new_group_thread.cs-CZ.text.erb]: Is this meant to be in English?

File: app/views/pages/changelog.cs-CZ.html.haml [hunk 2]
```diff
@@ -0,0 +1,265 @@
+#changelog
+  %h1 Co je nového?
+  %p
+    Toto je přehled nedávných změn na tété stránce.
+    Kompletní detailní historii vývoje můžete sledovat na
+    = t(".github_history_link_html")
+  %article
```
Comment (nikolai-b) [inline app/views/pages/changelog.cs-CZ.html.haml]: I think we should just split body of `changelog.html.haml` so https://github.com/cyclestreets/cyclescape/blob/master/app/views/pages/changelog.html.haml#L7 and here would become:
```ruby
  = render "changes"
```
and then move all everything below this line into `app/views/pages/_changes.html.haml`.  Does that make sense?

**Comment by nikolai-b** [comment]
Looks good.  If you can make the changes I suggested then we can merge it in.  Thanks.

**Comment by PetrDlouhy** [comment]
OK, I made aditional changes - extended the translation and dealt with the changelog with a bit different approach. I moved the introduction strings to the locale files.

When I will have enough capacity to translate the changelog, I will make czech version of the file once again.

File: config/locales/cs-CZ.yml [hunk 3]
```diff
@@ -714,9 +841,21 @@
...
       text_message: "Text"
+      street_view_message: Street View
   pages:
     changelog:
-      github_history_link_html: "<a href='https://github.com/cyclestreets/cyclescape/commits/master/'>GitHub</a>."
+      github_history_link_html: "<a href='https://github.com/cyclestreets/cyclescape/commits/master/'>GitHub</a>u."
+      header: "Co je nového?"
+      introduction: "Toto je přehled nedávných změn na tété stránce.
+    Kompletní detailní historii vývoje můžete sledovat na"
+>>>>>>> 168d672... translate header of changelog through locale
```
Comment (nikolai-b) [inline config/locales/cs-CZ.yml]: This line has been added in error.  Can you remove it please?

File: config/locales/en-GB.yml [hunk 4]
```diff
@@ -649,9 +649,9 @@
       new_user_location_issue_thread:
         subject: "[Cyclescape] New thread started on issue \"%{issue_title}\""
       group_request_confirmed:
-        subject: "[Cyclescape] New Group \"%{group_name}}\" has been created"
+        subject: "[Cyclescape] New Group \"%{group_name}\" has been created"
```
Comment (nikolai-b) [inline config/locales/en-GB.yml]: Thanks for spotting (and fixing) this!

**Comment by nikolai-b** [comment]
@PetrDlouhy thanks, there is just one line you need to remove (looks like a git merge conflict line), then we are good to merge.  Cheers.

**Comment by mvl22** [comment]
@nikolai-b Are you able to resolve the merge conflict yourself so we can get this in?

**Comment by PetrDlouhy** [comment]
I am on vacation and I will be able to solve this next week.

**Comment by nikolai-b** [comment]
Thanks @PetrDlouhy, as this isn't urgent then I think it makes sense to wait till your back.

**Comment by PetrDlouhy** [comment]
I have fixed the line and merged with current master. I hope that now is everything OK.

**Comment by mvl22** [comment]
Thanks Petr!

Now merged in, and on the live cyclescape.org server.

Welcome back from holiday, @nikolai-b!

**Pull request merged**